src/modules/postgres/setup/16-05-2016.update.sql

```diff
@@ -0,0 +1,46 @@
+UPDATE users SET identities = '{}' WHERE identities IS NULL;
+UPDATE items SET parents = '{}' WHERE parents IS NULL;
+UPDATE rooms SET name = '' WHERE name IS NULL;
+UPDATE threads SET name = '' WHERE name IS NULL;
+UPDATE threads SET body = '' WHERE body IS NULL;
+UPDATE texts SET body = '' WHERE body IS NULL;
+UPDATE notes SET count = DEFAULT WHERE count IS NULL;
+UPDATE notes SET data = DEFAULT WHERE data IS NULL;
+
+ALTER TABLE contacts ALTER COLUMN createtime SET NOT NULL;
+ALTER TABLE contacts ALTER COLUMN createtime SET DEFAULT extract(epoch from now());
+
+ALTER TABLE users ALTER COLUMN identities SET NOT NULL;
+ALTER TABLE users ALTER COLUMN createtime SET NOT NULL;
+ALTER TABLE users ALTER COLUMN tags SET NOT NULL;
```

Comment:
there are some users having tags and createtime null value. 
ERROR:  column "createtime" contains null values